common/src/main/scala/codacy/utils/FileHelpers.scala

```diff
@@ -4,31 +4,31 @@ import java.net.{URI, URL, URLClassLoader}
 
 import better.files._
 
-import scala.util.{Failure, Success, Try}
-
 import scala.collection.JavaConversions._
 
 object FileHelpers {
 
-  implicit class UrlExtension(val value:URL) extends AnyVal{
+  implicit class UrlExtension(val value: URL) extends AnyVal {
     def toFile = fileFromUri(value.toURI)
   }
 
-  private def fileFromUri(uri:URI):File = {
-    Try(File(uri)) match{
-      case Success(file) => file
-      case Failure(e:java.nio.file.FileSystemNotFoundException) =>
-        val env = new java.util.HashMap[String,String]
+  private def fileFromUri(uri: URI): File = {
+    def createFileSystemIfNotExists(): Unit = {
+      try {
+        val env = new java.util.HashMap[String, String]
         env.put("create", "true")
-        Try(java.nio.file.FileSystems.newFileSystem(uri, env))
-          .recover{ case _:java.nio.file.FileSystemAlreadyExistsException => () } match{
-          case Success(_) => ()
-          case Failure(err) => throw err
-        }
+        java.nio.file.FileSystems.newFileSystem(uri, env)
+      } catch {
+        case _: java.nio.file.FileSystemAlreadyExistsException =>
+      }
+    }
 
+    try {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider using util.Try instead of a try-catch block](https://app.codacy.com/gh/codacy/codacy-scalameta/pullRequest?prid=5195080)